moler/device/abstract_device.py

```diff
@@ -43,6 +46,27 @@ def name(self, value):
         :return: None
         """
 
+    @property
+    @abc.abstractmethod
+    def external_name(self):
+        """
+        Getter of device alias name. If you clone devices and forget them then if you want to create with already used
+        name then device will be created with different name but alias name will be as you want.
+
+        :return: String with the device alias name.
+        """
+
+    @external_name.setter
+    @abc.abstractmethod
+    def external_name(self, value):
+        """
+        Setter for device alias name. If you clone devices and forget them then if you want to create with already used
+        name then device will be created with different name but alias name will be as you want.
```

Comment:
Same comment as above - just `Setter for publicly used device name`

Reply:
Done